libs/core/algorithms/tests/unit/container_algorithms/foreach_tests.hpp

```diff
@@ -348,8 +348,8 @@ void test_for_each_sender(ExPolicy&& p, IteratorTag)
     auto rng = hpx::util::make_iterator_range(
         iterator(std::begin(c)), iterator(std::end(c)));
     auto f = [](std::size_t& v) { v = 42; };
-    auto result = ex::just(rng, f) |
-        hpx::ranges::for_each(std::forward<ExPolicy>(p)) | tt::sync_wait();
+    auto result = hpx::get<0>(*(ex::just(rng, f) |
+        hpx::ranges::for_each(std::forward<ExPolicy>(p)) | tt::sync_wait()));
     HPX_TEST(result == iterator(std::end(c)));
```

Comment:
Adding * in HPX_TEST(*result == iterator(std::end(c))) doesn't get rid of the errors. 
I think for the sync_wait(), * is not enough to get the one tuple.
So I tried to pull out the value first as above(with get<0>(*...)).
Please correct me if I did it wrong.